src/qt/bitcoin.cpp

```diff
@@ -60,9 +61,10 @@ Q_IMPORT_PLUGIN(QCocoaIntegrationPlugin);
 #endif
 #endif
 
-// Declare meta types used for QMetaObject::invokeMethod
+// Make types (not aliases) available to QVariant and other template-based Qt functions.
+// Additionally qRegisterMetaType() calls are required to make them available to
+// non-template based Qt functions, including those that use Qt::QueuedConnection.
```

Comment:
nit, could mention env `QT_FATAL_WARNINGS=1` as a way to identify missing registrations.

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/61#issuecomment-704127075).